packages/libs/src/sdk/config/production.ts

```diff
@@ -9,247 +9,303 @@ export const servicesConfig: ServicesConfig = {
     {
       "endpoint": "https://audius-discovery-2.cultur3stake.com",
-      "delegateOwnerWallet": "0x7c125128B0917bDE12e6A0eDde8F7675d4ADF408"
+      "delegateOwnerWallet": "0x7c125128B0917bDE12e6A0eDde8F7675d4ADF408",
+      "ownerWallet": "0x2168990Cd51c7C7DdE4b16Ac4fe7dbA269768990"
     },
     {
       "endpoint": "https://blockdaemon-audius-discovery-08.bdnodes.net",
-      "delegateOwnerWallet": "0x8464c88502925a0076c381962F8B70b6EC892861"
+      "delegateOwnerWallet": "0x8464c88502925a0076c381962F8B70b6EC892861",
+      "ownerWallet": "0x091D2190e93A9C09f99dA05ec3F82ef5D8aa4a07"
     },
     {
       "endpoint": "https://audius-metadata-5.figment.io",
-      "delegateOwnerWallet": "0x69cfDc1AB75384f077E4E48cf0d6483C8fB9B8A2"
+      "delegateOwnerWallet": "0x69cfDc1AB75384f077E4E48cf0d6483C8fB9B8A2",
+      "ownerWallet": "0x700a11aE95E34fBC769f8EAD063403987Bd0C502"
     },
     {
       "endpoint": "https://dn1.stuffisup.com",
-      "delegateOwnerWallet": "0xAA29e93f4008D977078957D8f041AEAeF7e1eeBc"
+      "delegateOwnerWallet": "0xAA29e93f4008D977078957D8f041AEAeF7e1eeBc",
+      "ownerWallet": "0x3E2Cd6d498b412Da182Ef25837F72355f8918BE9"
     },
     {
       "endpoint": "https://audius-discovery-1.theblueprint.xyz",
-      "delegateOwnerWallet": "0xEDe07aCa59815fbaa75c4f813dCDD1390D371071"
+      "delegateOwnerWallet": "0xEDe07aCa59815fbaa75c4f813dCDD1390D371071",
+      "ownerWallet": "0x68f656d19AC6d14dF209B1dd6E543b2E81d53D7B"
     },
     {
       "endpoint": "https://audius-discovery-2.theblueprint.xyz",
-      "delegateOwnerWallet": "0xCF3f359BfdE7bcAfE4bc058B6DFae51aBe204aB4"
+      "delegateOwnerWallet": "0xCF3f359BfdE7bcAfE4bc058B6DFae51aBe204aB4",
+      "ownerWallet": "0x68f656d19AC6d14dF209B1dd6E543b2E81d53D7B"
     },
     {
       "endpoint": "https://audius-discovery-3.theblueprint.xyz",
-      "delegateOwnerWallet": "0x8449169096550905B903b6803FB3b64285112603"
+      "delegateOwnerWallet": "0x8449169096550905B903b6803FB3b64285112603",
+      "ownerWallet": "0x68f656d19AC6d14dF209B1dd6E543b2E81d53D7B"
     },
     {
       "endpoint": "https://audius-discovery-4.theblueprint.xyz",
-      "delegateOwnerWallet": "0x16e8DF288BF5DcD507615A715A2a6155F149a865"
+      "delegateOwnerWallet": "0x16e8DF288BF5DcD507615A715A2a6155F149a865",
+      "ownerWallet": "0x68f656d19AC6d14dF209B1dd6E543b2E81d53D7B"
     },
     {
       "endpoint": "https://audius.w3coins.io",
-      "delegateOwnerWallet": "0x1cBC80a8A4A6fd5793e535B758A71f332DEc1551"
+      "delegateOwnerWallet": "0x1cBC80a8A4A6fd5793e535B758A71f332DEc1551",
+      "ownerWallet": "0x53B954C36b8f9f7AefdC9791280b95232C255D1b"
     },
     {
       "endpoint": "https://audius-nodes.com",
-      "delegateOwnerWallet": "0xE83699015c8eb793A0678eA7dC398ac58f7928c4"
+      "delegateOwnerWallet": "0xE83699015c8eb793A0678eA7dC398ac58f7928c4",
+      "ownerWallet": "0xE83699015c8eb793A0678eA7dC398ac58f7928c4"
+    },
+    {
+      "endpoint": "https://audius-dn1.tikilabs.com",
```

Comment:
@sliptype seems like we should regenerate these from time to time? not a huge priority, maybe on every minor release of protocol or even on a cadence of some sort